### PR TITLE
[hi] localize canary_deployment.md

### PR DESCRIPTION
**Pull request by Aayush987**
Hi,
This Pr is Hindi localization of content/hi/canary_deployment.md page.

ref : #270 
/lang hi

@anubha-v-ardhan @Garima-Negi and everyone plz review.

### CODE REVIEW

File: content/hi/canary_deployment.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: कैनरी डिप्लॉयमेंट
+status: completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+कैनरी डिप्लॉयमेंट (Canary_Deployment), एक डिप्लॉयमेंट रणनीति है जिसकी शुरुआत होती है दो एनवायरनमेंट से : एक लाइव ट्रैफिक के साथ और दूसरा जिसमे अपडेटेड कोड हो बिना लाइव ट्रैफिक के। ट्रैफिक को धीरे-धीरे एप्लीकेशन के मूल संस्करण से अपडेटेड संस्करण में लाया जाता है। यह 1% लाइव ट्रैफ़िक, फिर 10%, 25%, इत्यादि को स्थानांतरित करके शुरू कर सकता है, जब तक कि सभी ट्रैफ़िक अपडेट किए गए संस्करण के माध्यम से नहीं चल रहे हों। संगठन उत्पादन में सॉफ़्टवेयर के नए संस्करण का परीक्षण कर सकते हैं, प्रतिक्रिया प्राप्त कर सकते हैं, त्रुटियों का निदान कर सकते हैं, और यदि आवश्यक हो तो स्थिर संस्करण में त्वरित रूप से रोलबैक कर सकते हैं।
+
+"कैनरी" शब्द "कोयला खदान में कैनरी" प्रथा को संदर्भित करता है जहां खनिकों को सुरक्षित रखने के लिए कैनरी पक्षियों को कोयला खदानों में ले जाया जाता था। यदि गंधहीन हानिकारक गैसें मौजूद होतीं, तो पक्षी मर जाता, और खनिक जानते थे कि उन्हें जल्दी से खाली करना होगा। इसी तरह, अगर अपडेट किए गए कोड में कुछ गलत हो जाता है, तो लाइव ट्रैफ़िक को मूल संस्करण में वापस "निकासी" कर दिया जाता है।
```
Comment (anubha-v-ardhan) [inline content/hi/canary_deployment.md:11]:
```suggestion
"कैनरी" शब्द "कोयला खदान में कैनरी" प्रथा को संदर्भित करता है जहां खनिकों को सुरक्षित रखने के लिए कैनरी पक्षियों को कोयला खदानों में ले जाया जाता था। यदि गंधहीन हानिकारक गैसें मौजूद होतीं, तो पक्षी मर जाता, और खनिक जानते थे कि उन्हें जल्दी से निकलना होगा। इसी तरह, अगर अपडेट किए गए कोड में कुछ गलत हो जाता है, तो लाइव ट्रैफ़िक को मूल संस्करण में वापस कर दिया जाता है।

```

File: content/hi/canary_deployment.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## समस्या
+
+कोई फर्क नहीं पड़ता कि परीक्षण रणनीति कितनी भी अच्छी हो ,उत्पादन के समय हमेशा कुछ बग मिल जाते हैं। 100% ट्रैफ़िक को ऐप के एक संस्करण से दूसरे संस्करण में ले जाने से अधिक प्रभावशाली असफलताए हो सकती हैं।
+
+## समाधान
+
+कैनरी डिप्लॉयमेंट संगठनों को महत्वपूर्ण ट्रैफ़िक को नए संस्करण में ले जाने से पहले यह देखने की अनुमति देता है कि नया सॉफ़्टवेयर वास्तविक दुनिया परिदृश्य में कैसे व्यवहार करता है। यह रणनीति संगठनों को नए डिप्लॉयमेंट के साथ समस्याओं के मामले में डाउनटाइम को कम करने और जल्दी से रोलबैक करने में सक्षम बनाती है। यह संपूर्ण उपयोगकर्ता अनुभव पर महत्वपूर्ण प्रभाव के बिना अधिक गहराई में उत्पादन अनुप्रयोग परीक्षण की भी अनुमति देता है।
```
Comment (anubha-v-ardhan) [inline content/hi/canary_deployment.md:19]:
```suggestion

कैनरी डिप्लॉयमेंट संगठनों को महत्वपूर्ण ट्रैफ़िक को नए संस्करण में ले जाने से पहले यह देखने की अनुमति देता है कि नया सॉफ़्टवेयर वास्तविक दुनिया परिदृश्य में कैसे व्यवहार करता है। यह रणनीति संगठनों को नए डिप्लॉयमेंट के साथ समस्याओं के मामले में डाउनटाइम को कम करने और जल्दी से रोलबैक करने में सक्षम बनाती है। यह संपूर्ण उपयोगकर्ता अनुभव पर विशेष प्रभाव के बिना अधिक गहराई से उत्पादन एप्लीकेशन परीक्षण की भी अनुमति देता है।

```

**Review by anubha-v-ardhan — COMMENTED** [review]
Thank you @Aayush987 
Please go through the suggestions and commit them if you agree.
Feel free to discuss if you find inaccuracy or feel like a change is uneccessary

File: content/hi/canary_deployment.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: कैनरी डिप्लॉयमेंट
+status: completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+कैनरी डिप्लॉयमेंट (Canary_Deployment), एक डिप्लॉयमेंट रणनीति है जिसकी शुरुआत होती है दो एनवायरनमेंट से : एक लाइव ट्रैफिक के साथ और दूसरा जिसमे अपडेटेड कोड हो बिना लाइव ट्रैफिक के। ट्रैफिक को धीरे-धीरे एप्लीकेशन के मूल संस्करण से अपडेटेड संस्करण में लाया जाता है। यह 1% लाइव ट्रैफ़िक, फिर 10%, 25%, इत्यादि को स्थानांतरित करके शुरू कर सकता है, जब तक कि सभी ट्रैफ़िक अपडेट किए गए संस्करण के माध्यम से नहीं चल रहे हों। संगठन उत्पादन में सॉफ़्टवेयर के नए संस्करण का परीक्षण कर सकते हैं, प्रतिक्रिया प्राप्त कर सकते हैं, त्रुटियों का निदान कर सकते हैं, और यदि आवश्यक हो तो स्थिर संस्करण में त्वरित रूप से रोलबैक कर सकते हैं।
```
[the remark this answers is not in the document]
Comment (Aayush987) [reply content/hi/canary_deployment.md:9]: Done but don't I have to leave line after line 7?

[the remark this answers is not in the document]
Comment (anubha-v-ardhan) [reply content/hi/canary_deployment.md:9]: Not necessary, markdown doesn't mind if you leave line or not

[the remark this answers is not in the document]
Comment (Aayush987) [reply content/hi/canary_deployment.md:9]: ok

[the remark this answers is not in the document]
Comment (Aayush987) [reply content/hi/canary_deployment.md]: Done

File: content/hi/canary_deployment.md (hunk 1 above)
Comment (Aayush987) [reply content/hi/canary_deployment.md]: Done

File: content/hi/canary_deployment.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+"कैनरी" शब्द "कोयला खदान में कैनरी" प्रथा को संदर्भित करता है जहां खनिकों को सुरक्षित रखने के लिए कैनरी पक्षियों को कोयला खदानों में ले जाया जाता था। यदि गंधहीन हानिकारक गैसें मौजूद होतीं, तो पक्षी मर जाता, और खनिक जानते थे कि उन्हें जल्दी से खाली करना होगा। इसी तरह, अगर अपडेट किए गए कोड में कुछ गलत हो जाता है, तो लाइव ट्रैफ़िक को मूल संस्करण में वापस "निकासी" कर दिया जाता है।
+
+## समस्या
+
+कोई फर्क नहीं पड़ता कि परीक्षण रणनीति कितनी भी अच्छी हो ,उत्पादन के समय हमेशा कुछ बग मिल जाते हैं। 100% ट्रैफ़िक को ऐप के एक संस्करण से दूसरे संस्करण में ले जाने से अधिक प्रभावशाली असफलताए हो सकती हैं।
```
[the remark this answers is not in the document]
Comment (Aayush987) [reply content/hi/canary_deployment.md]: Done

File: content/hi/canary_deployment.md (hunk 2 above)
Comment (Aayush987) [reply content/hi/canary_deployment.md]: Done

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!
Thank you @Aayush987

**Review by jayesh-srivastava — APPROVED** [review]

**Comment by jayesh-srivastava** [comment]
lgtm

**Pull request merged**